fixtures/ios_app_with_framework_and_missing_resources/App/Config/App-Info.plist

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```

Comment:
I would not add a fixture for this. If we end up with too many fixtures it creates confusion IMO